File: src/domain/entities/condition.entity.ts
export class ConditionEntity {
    constructor(
        public id: number,
        public name: string,
        public description: string,
    ){}

    public static fromObject(object: {[key: string] : any}) : ConditionEntity {
        const { id, name, description } = object;
        return new ConditionEntity(
            id,
            name,
            description
        );
    }
}

File: python/app/domain/condition_entity.py
class ConditionEntity:
    def __init__(self, id: int, name: str, description: str):
        self.id = id
        self.name = name
        self.description = description

    @staticmethod
    def from_dict(data: dict) -> 'ConditionEntity':
        return ConditionEntity(
            id=data.get('id'),
            name=data.get('name'),
            description=data.get('description')
        )

    def __repr__(self):
        return f"ConditionEntity(id={self.id}, name='{self.name}', description='{self.description}')"
